scripts/qa_crowbarsetup.sh

```diff
@@ -3405,11 +3405,15 @@ function onadmin_proposal
     fi
     local proposal
     for proposal in nfs_client pacemaker database rabbitmq keystone swift \
-        ceph monasca glance cinder neutron nova `horizon_barclamp` \
-        ceilometer heat manila trove barbican magnum sahara murano \
-        aodh tempest; do
+        ceph monasca glance cinder neutron ; do
```

Comment:
It is split into two sets, but it's pre-nova and post-nova, because it depends on the oncontroller function working properly, and the oncontroller function does not work until after the nova barclamp has been deployed. I think that should probably change but I didn't want this pull request to make too many crazy changes at once.